Gemfile

```diff
@@ -27,4 +27,8 @@ else
   gem 'puppet', :require => false
 end
 
+if RUBY_VERSION < "1.9"
+  gem 'activesupport', '<=3.1.0', :require => false
```

Comment:
Gems in the Gemfile are only usable for running tests. They do not get installed on agents and therefore can't be used in types and providers. I get this error when trying to create a user:

```
Error: Could not autoload puppet/type/mongodb_user: Could not autoload puppet/provider/mongodb_user/mongodb: no such file to load -- active_support/ordered_hash
```